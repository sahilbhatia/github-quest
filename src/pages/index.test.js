import React from "react";
import Index from "./index";
import renderer from "react-test-renderer";

test("test case for index page", () => {
  const component = renderer.create(<Index />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
