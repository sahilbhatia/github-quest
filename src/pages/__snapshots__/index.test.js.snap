// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test case for index page 1`] = `
<div>
  <h1
    className="text-center"
  >
    welcome to Github-Quest
  </h1>
  <br />
  <div
    className="d-flex justify-content-around"
  >
    <h3
      className="text-primary border rounded border-dark p-2"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <u>
        User List
      </u>
    </h3>
    <h3
      className="text-primary border rounded border-dark p-2"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <u>
        Project List
      </u>
    </h3>
    <h3
      className="text-primary rounded border-dark border p-2"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <u>
        Repository List
      </u>
    </h3>
    <h3
      className="text-primary rounded border-dark border p-2"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <u>
        Errors
      </u>
    </h3>
  </div>
</div>
`;
